From 99a8c0b5e3f302bf522c60f1de3c5e710857130f Mon Sep 17 00:00:00 2001
From: Liu Ying <victor.liu@nxp.com>
Date: Fri, 1 Jun 2018 16:21:49 +0800
Subject: [PATCH 3897/5242] MLK-18477-3 arm64: fsl-imx8qm.dtsi: Remove dpu
 assigned-clock* properties

commit  e3f09ba8e09457cd317760d6581d1dc5774989ca from
https://source.codeaurora.org/external/imx/linux-imx.git

The dpu driver may properly handle display clock parent selection now.
Thus, let's remove the assigned-clocks and assigned-clock-parents
device tree properties from the dpu device tree nodes.

Signed-off-by: Liu Ying <victor.liu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qm-device.dtsi      |    8 --------
 1 file changed, 8 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
index 9b56b51..cea8956 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
@@ -1676,10 +1676,6 @@
 			 <&clk IMX8QM_DC0_DISP1_CLK>;
 		clock-names = "pll0", "pll1", "bypass0",
 			      "disp0_sel", "disp1_sel", "disp0", "disp1";
-		assigned-clocks = <&clk IMX8QM_DC0_DISP0_SEL>,
-				  <&clk IMX8QM_DC0_DISP1_SEL>;
-		assigned-clock-parents = <&clk IMX8QM_DC0_PLL0_CLK>,
-					 <&clk IMX8QM_DC0_PLL1_CLK>;
 		power-domains = <&pd_dc0_pll1>;
 		fsl,dpr-channels = <&dpr1_channel1>, <&dpr1_channel2>,
 				   <&dpr1_channel3>, <&dpr2_channel1>,
@@ -2144,10 +2140,6 @@
 			 <&clk IMX8QM_DC1_DISP1_CLK>;
 		clock-names = "pll0", "pll1", "bypass0",
 			      "disp0_sel", "disp1_sel", "disp0", "disp1";
-		assigned-clocks = <&clk IMX8QM_DC1_DISP0_SEL>,
-				  <&clk IMX8QM_DC1_DISP1_SEL>;
-		assigned-clock-parents = <&clk IMX8QM_DC1_PLL0_CLK>,
-					 <&clk IMX8QM_DC1_PLL1_CLK>;
 		power-domains = <&pd_dc1_pll1>;
 		fsl,dpr-channels = <&dpr3_channel1>, <&dpr3_channel2>,
 				   <&dpr3_channel3>, <&dpr4_channel1>,
-- 
1.7.9.5

